### ISSUE TITLE
v2.5.1 package.json points to coffeescript file

### ISSUE THREAD

**Issue by cunneen** [softwarerero/meteor-accounts-t9n#194]
Attempting to use this package without coffeescript results in a server error:

```
=> Exited with code: 1
W20190618-13:27:45.393(8)? (STDERR) /mymeteorprogram/node_modules/meteor-accounts-t9n/t9n.coffee:1
W20190618-13:27:45.394(8)? (STDERR) (function (exports, require, module, __filename, __dirname) { Meteor?.startup ->
W20190618-13:27:45.394(8)? (STDERR)                                                                      ^
W20190618-13:27:45.394(8)? (STDERR) 
W20190618-13:27:45.394(8)? (STDERR) SyntaxError: Unexpected token .
W20190618-13:27:45.394(8)? (STDERR)     at createScript (vm.js:80:10)
(...)
W20190618-13:27:45.395(8)? (STDERR)     at t9n.coffee (packages/modules.js:318:8)
```

It seems that v2.5.1 of the package was accidentally published with `package.json` referring to `t9n.coffee` instead of `build/t9n.js` :

https://github.com/softwarerero/meteor-accounts-t9n/commit/5489ed6fa17f561e6e3c55741b421e5a48f40096#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12

**Comment by cunneen** [comment]
Downgrading to v2.3.1 fixes it for me :

```
meteor npm install meteor-accounts-t9n@~2.3.1 --save
```

**Comment by softwarerero** [comment]
@cunneen Thanks for reporting, should be fixed with 2.5.2.
